CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## v6.3.2 - 2018-06-26
 ### Fixed
+* Fixed broken rendering of equipment with a requirement ([#1730](https://github.com/YaleSTC/reservations/issues/1730)).
```

Comment:
Do you mind putting this at the end of the section? My OCD self likes to put the CHANGELOG in numerical order 😛 . Thanks!